### PR TITLE
Added Guided Practice and Independent Practice to css.md

### PR DESCRIPTION
**Pull request by pydevsg**
Fixed Issue #698

### CODE REVIEW

File: css/css.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ About 4.5 hours
 - Describe the hierarchy of specificity for CSS selectors
 - Describe the different parts of the "Box" model
 - Effectively use the `display` property
+- Changing and Removing styles from CSS in a webpage.
```
Comment (Kundan28) [inline css/css.md]: You can add full stop after above 3 points also.

Comment (Kundan28) [reply css/css.md]: And try to remove one white space after `-` in the line number 23.

File: css/css.md [hunk 2]
```diff
@@ -76,11 +77,17 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
 
 [Click here to access the instructions for the CSS portion of the recipe-page project](https://github.com/Techtonica/curriculum/blob/master/projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
 
+While removing *style* tag , you should check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents from the webpage.
+
+Tutorials for changing and removing style tags is being pprovided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```
Comment (Kundan28) [inline css/css.md]: The sideshow is not following the formatting guidelines.

**Comment by Kundan28** [comment]
@alodahl In the slides the HTML code is in the font style `Consolas`. So we have to change that too to Lato ?

**Comment by pydevsg** [comment]
I have resol;ved all the issue @Kundan28 , please do have a review.

**Comment by Kundan28** [comment]
And please resolve all the comments I have mentioned in the slides comment.

**Comment by saberprashant** [comment]
Hi @pydevsg, how are you doing with this one? Are you facing any difficulty in making the changes which @Kundan28 suggested? If yes, then please let us know, we'll help you 😄

**Comment by Kundan28** [comment]
> Good work 😄 , but please make the changes which @Kundan28 suggested.

And the changes in the slideshow too.

**Comment by pydevsg** [comment]
@Kundan28 I have made all the changes in the slideshow, do I need to remove >CSS walkline video ?
The new video needs to be made in mp4 format and that is not part of this issue.

**Comment by Kundan28** [comment]
> @Kundan28 I have made all the changes in the slideshow, do I need to remove _CSS walkline video_ ?

Some more changes are to be made.

File: css/css.md [hunk 3]
```diff
@@ -76,6 +77,9 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
 
 [Click here to access the instructions for the CSS portion of the recipe-page project](https://github.com/Techtonica/curriculum/blob/master/projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
 
+While removing *style* tag , you should check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents from the webpage.
```
Comment (alodahl) [inline css/css.md]: Why are they checking this? Can you explain more?

File: css/css.md [hunk 4]
```diff
@@ -76,6 +77,9 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
...
+
+Tutorials for changing and removing style tags is being provided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```
Comment (alodahl) [inline css/css.md]: is being provided -> are provided

File: css/css.md (hunk 3 above)
Comment (pydevsg) [reply css/css.md]: Yes @alodahl this part is provided as because to remove style tags you need to get the inline css from the webpage, which is essential.

Comment (alodahl) [reply css/css.md]: Can you explain it for the apprentices with another sentence or 2?

Comment (alodahl) [reply css/css.md]: also, I hope you mean the style tag and not the style property

Comment (pydevsg) [reply css/css.md]: Yes, I will @alodahl

**Comment by pydevsg** [comment]
Please do review @alodahl and @Kundan28.

File: css/css.md [hunk 5]
```diff
@@ -51,11 +52,11 @@ About 4.5 hours
 
 ### Lesson
 
-[CSS (video walkthrough of slides)](https://drive.google.com/file/d/1IyAozbB3BAuFXdAZH1tu0kr-eL3El0Cn/view?usp=sharing)
+[CSS (video walkthrough of slides)](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```
Comment (alodahl) [inline css/css.md]: this is not a video.  please change it back.

File: css/css.md [hunk 6]
```diff
@@ -76,6 +77,9 @@ The activity for the HTML, CSS, Bootstrap and jQuery lessons is a project that a
 
 [Click here to access the instructions for the CSS portion of the recipe-page project](https://github.com/Techtonica/curriculum/blob/master/projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
 
+Check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents while removing *style* tag from the webpage.
+
+Tutorials for changing and removing style tags are provided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
```
Comment (alodahl) [inline css/css.md]: Can you add a ### Challenge section after this and ask them to remove a style tag from a real website themselves?  Give them 2 or 3 links to choose from where there's an obvious style tag in the head.

**Comment by alodahl** [comment]
Waiting on your last few changes so we can merge and give you your points, @pydevsg !

File: css/css.md (hunk 5 above)
Comment (pydevsg) [reply css/css.md]: Done !

File: css/css.md (hunk 6 above)
Comment (pydevsg) [reply css/css.md]: I have added one as there are not many such websites. Hopefully, the content for CSS will help solve any challenge as it is very good and have been reviewed carefully..

**Comment by pydevsg** [comment]
Hope so, now it is ready to be merged.

File: css/css.md (hunk 6 above)
Comment (Kundan28) [reply css/css.md]: There are many websites... @pydevsg 
https://developers.google.com/
https://events.google.com/io/
And many more...

Comment (alodahl) [reply css/css.md]: I’m not saying find tutorials, although that link you found is a good resource. I’m saying find learners regular websites where they can remove an obvious style tag and see a difference. Then tell them to try it themselves in independent practice.

File: css/css.md (hunk 5 above)
Comment (alodahl) [reply css/css.md]: This is still not a video, sorry. Always double check your links!

File: css/css.md (hunk 6 above)
Comment (alodahl) [reply css/css.md]: These are great, but I had to remove linked stylesheets from amazon.in.  Can you rephrase this sentence to refer to style and link tags, and not mention inline?

**Comment by pydevsg** [comment]
@alodahl Can you please share me the video link to be added at that place, I will make the necessary steps.

**Comment by Kundan28** [comment]
> @alodahl Can you please share me the video link to be added at that place, I will make the necessary steps.

The link before was of the video...
Here's the link (https://drive.google.com/file/d/1IyAozbB3BAuFXdAZH1tu0kr-eL3El0Cn/view?usp=sharing)

**Comment by alodahl** [comment]
@pydevsg The video is fine now, thanks.  The video actually has 2 sections, so it is the same video.  If you think it would be better to note the time Part 2 of the video starts, that would be awesome.  Besides that, just fix line 80 and you are done.

**Comment by pydevsg** [comment]
@alodahl , is there anything else which needs to be changed?
